benchopt/plotting/__init__.py

```diff
@@ -56,6 +53,12 @@ def plot_benchmark(fname, benchmark, kinds=None, display=True, plotly=False,
     if kinds is not None and len(kinds) > 0:
         config["plots"] = kinds
```

Comment:
Don't you need to handle the case where `kinds` is `None`, as defautl config is also `None`?